cli/js/testing.ts

```diff
@@ -179,6 +183,13 @@ interface TestEventEnd {
   results: TestResult[];
 }
 
+async function date(): Promise<number> {
+  if ((await permissions.query({ name: "hrtime" })).state == "granted") {
+    return performance.now();
+  }
```

Comment:
That's 2 ops dispatched to get high precision timing, most of the time it will not be available.